src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -83,7 +83,26 @@ VALUES
     ('02438278-ab93-4689-b123-7ac4e78f59fe', 'smsWebEmail_1000'    , 'responses per site'           , '1000 SMS/web/email responses per site'),
     ('67e1c174-8443-4883-b51f-43c297fa9c08', 'smsWebEmail_1500'    , 'responses per site'           , '1500 SMS/web/email responses per site'),
     ('1a36f980-76a4-4b3d-b36d-46afc6655124', 'dataMigration'       , 'data migrations'              , 'per data migration'),
-    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service');
+    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service'),
+    ('f24afc45-971d-41fa-9beb-6f577e987c6d', 'patients_1-20k'      , 'patients'                     , 'per patient for 1-19,999 patients'),
+    ('0fb82859-aa6c-4db5-b513-c776c7278aca', 'patients_20-150k'    , 'patients'                     , 'per patient 20,000 - 149,999 patients'),
+    ('7b2c46d4-e5c1-4315-a05e-021e43d485ee', 'patients_150-250k'   , 'patients'                     , 'per patient 150,000 - 249,999 patients'),
+    ('d37f63e0-0bb2-4703-9f86-f22057231a00', 'patients_250k'       , 'patients'                     , 'per patient 250,000 patients & above'),
+    ('950365df-456d-4c72-bc6f-74a0f7535728', 'patientRegistered'   , 'patients'                     , 'per registered patient'),
+    ('2c25c251-168d-4f14-a637-3d09fe7440b7', 'onlineConsultationGP', 'consultations'                , 'per online consultation non GP practices'),
+    ('edb80481-d99a-4d25-8d58-da6881b453f7', 'videoConsultationGP' , 'consultations'                , 'per video consultation non GP practices'),
+    ('a2441010-cfd0-4a0a-bbf7-e73e279b4298', 'referralEps_1-5k'    , 'referral episodes'            , 'per referral episode 1-5000 episodes'),
+    ('c4bfbaf0-f311-47d2-b02c-7ee76b9f779f', 'referralEps_5-20k'   , 'referral episodes'            , 'per referral 5001 - 20,000 episodes'),
+    ('2f59e518-e211-44bb-b9b3-0fc518f4f46c', 'referralEps_20-50k'  , 'referral episodes'            , 'per referral 20,001 - 50,000 episodes'),
+    ('5c863dc7-b17f-43aa-997a-e5d1226525d8', 'referralEps_50k'     , 'referral episodes'            , 'per referral 50,0001 or more episodes'),
+    ('3d49fdd4-a949-40ab-adea-f541a6dbe034', 'trainingSession_360' , 'training sessions'            , 'per 4 x 90 min online training sessions'),
+    ('035c6c00-773b-47f7-af61-4eb60a1ce639', 'language'            , 'languages'                    , 'per Language'),
```

Comment:
units are usually lower case, so should be 'per language'.

Reply:
updated unit to lowercase